cdap-docs/developer-manual/source/getting-started/building-apps.rst

```diff
@@ -125,7 +125,7 @@ Once CDAP is started, you can deploy an application using an example JAR by any
 
     examples/|example-dir|/target/|example|-|release|.jar
 
-- In the :ref:`new beta CDAP UI <cdap-ui-beta>`,  use the green "plus" button |---| found on either
+- In the CDAP UI, use the green "plus" button |---| found on either
   the |home-page|_ or |management-page|_ pages |---| to bring up the *Hub/Resource
```

Comment:
oh gosh this page is outdated too. @albertshau let me know if you want me to push a commit.

Reply:
go for it